charts/castai-hosted-model/child-charts/vllm/templates/deployment.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "vllm.fullname" . }}
+  labels:
+    {{- include "vllm.labels" . | nindent 4 }}
+spec:
+  replicas: {{ .Values.replicaCount }}
+  selector:                                                                                                                                  
+    matchLabels:
+      {{- include "vllm.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "vllm.labels" . | nindent 8 }}
+        {{- with .Values.podLabels }}
+        {{- toYaml . | nindent 8 }}
+        {{- end }}
+    spec:
+      containers:
+        - name: "vllm"
+          image: "{{ required "Required value 'image.repository' must be defined" .Values.image.repository }}:{{ required "Required value 'image.tag' must be defined" .Values.image.tag }}"
+          imagePullPolicy: IfNotPresent
+          command: ["/bin/sh", "-c", "vllm", "serve"]
+          args:
+            - {{ .Values.model | quote }}
+            - "--trust-remote-code"
+            - "--gpu-memory-utilization=1"
+            - "--task=generate"
+            - "--dtype=half"
+            - "--kv-cache-dtype=fp8"
+            - "--enable-chunked-prefill=False"
```

Comment:
Is there any reason to disable this? From what I've read - enabling this provides better GPU utilization.

Reply:
No reason. Enabled it!